eng/pipelines/templates/stages/vmr-verticals.yml

```diff
@@ -995,3 +995,77 @@ stages:
             - Windows_x64
             - Windows_x86
             - Windows_arm64
+
+        # Source Build Leg
```

Comment:
Is there value in embedding the SB leg and SB validation leg in new jobs template?  The benefit is to ensure they both get done and help in ensuring parameter consistency between the two - e.g. containers, naming, etc.  This would make it easier to add new SB legs.

Reply:
Fixed with https://github.com/dotnet/sdk/pull/48327/commits/b8206dde44e2e2bb0b2a767553e4d0ee18aeaf4c